proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -158,6 +158,20 @@ message SearchTxsResult {
   repeated TxResponse txs = 6;
 }
 
+// UnconfirmedTxsResult defines a structure for querying mempool txs
+message UnconfirmedTxsResult {
```

Comment:
Why is this message specified in proto? It doesn't seem to be marshaled as bytes anywhere.